app/css/views/add-tree.css

```diff
@@ -13,4 +13,66 @@
 }
 #submit-add-tree:hover {
 	text-decoration: none;
+}
+
+input {
```

Comment:
Try to avoid using styling for HTML elements like this. Instead try using class name selectors like this:
```css
.form-field {
/* a generic form element like <input>, <select>, <textarea>, etc. */
}
.form-button {
/* a button inside of a form (like to send the form data) */
}
.form-row {
/* a row inside of a form (block-level wrapper element) */
}
.form-group {
/* a group of fields in a form */
}
```

The problem with using selectors for HTML elements is that you will have a harder time overriding the rules later (especially as a project grows in complexity).